blocks/live-now/live-now.js

```diff
@@ -126,27 +126,54 @@ function buildModal(title, thumbnailPath, url) {
   const playSvg = `${window.hlx.codeBasePath}/icons/video_play.svg`;
 
   let cidCode;
-  const locale = navigator.language;
-  const language = locale.split('-')[0];
+  let modalTitle;
+  let modalHeader;
+  let joinMsg;
+  const windowUrl = window.location.href;
+  // eslint-disable-next-line no-useless-escape
+  const langRegex = /https:\/\/[^\/]+\/([a-z]{2})(\/|$)/;
+  const langMatches = windowUrl.match(langRegex);
+  const language = langMatches ? langMatches[1] : 'en';
   const cidCodes = {
-    en: '?intcid=wbw_xpl_overlay_en_ext',
-    es: '?intcid=wbw_xpl_overlay_es_ext',
-    fr: '?intcid=wbw_xpl_overlay_fr_ext',
-    ar: '?intcid=wbw_xpl_overlay_ar_ext',
+    en: '?intcid=wbw_xpl_liveoverlay_en_ext',
+    es: '?intcid=wbw_xpl_liveoverlay_es_ext',
+    fr: '?intcid=wbw_xpl_liveoverlay_fr_ext',
+    ar: '?intcid=wbw_xpl_liveoverlay_ar_ext',
+  };
+  const modalTitles = {
+    en: 'Live Now',
```

Comment:
@mdickson-adbe Please use placeholders for translation.

Reply:
I've added placeholder files for FR and AR which were missing. Also refactored the modal to use those data sources.